engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/PassPersistance.java

```diff
@@ -67,6 +68,7 @@
 @Persistable(clazz = TypeInference.class, id = 1280)
 @Persistable(clazz = FramePointerAnalysis$.class, id = 1281)
 @Persistable(clazz = TailCall.TailPosition.class, id = 1282)
+@Persistable(clazz = CachePreferences.class, id = 1284)
```

Comment:
- Adding this class without registering `CachePreferences.Kind` for persistance is causing #11858